### PR TITLE
Task #49261 [Admin] List All Room View

### PR DESCRIPTION
**Pull request by anhlh-3049**
## Related Tickets
- https://edu-redmine.sun-asterisk.vn/issues/49261

## WHAT (optional)
- List All RoomView

## Evidence (Screenshot or Video)
Screenshot_8

### CODE REVIEW

**Comment by anhlh-3049** [comment]
ready

File: app/controllers/admin/room_views_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Admin::RoomViewsController < Admin::AdminController
+  def index
+    @pagy, @roomviews = pagy RoomView.all
```
Comment (ngocvt-0484) [inline app/controllers/admin/room_views_controller.rb]:
```suggestion
    @pagy, @room_views = pagy RoomView.all
```

File: config/routes.rb [hunk 2]
```diff
@@ -12,5 +12,6 @@
 
     resources :dash_broads, only: :index
     resources :rooms
+    resources :room_views
```
Comment (ngocvt-0484) [inline config/routes.rb]: only :index đi

**Comment by anhlh-3049** [comment]
ready

File: app/views/admin/room_views/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            <th><%= t ".name" %></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @room_views.each_with_index do |room_view, index| %>
+            <tr>
+              <td><%= index + 1 %></td>
+              <td><%= link_to room_view.name, "#" %></td>
+            </tr>
+          <% end %>
```
Comment (manhnt-0960) [inline app/views/admin/room_views/index.html.erb]: dùng partial collection cho performance tốt hơn nhé

File: config/locales/vi.yml [hunk 4]
```diff
@@ -49,6 +49,13 @@ vi:
         provide_title: "Quản lý phòng"
         create_btn: "Tạo phòng mới"
         view_all_btn: "Xem tất cả các phòng"
+    room_views:
+      index:
+        provide_title: "Quang cảnh phòng"
+        title: "Quang cảnh phòng"
+        id: "ID"
```
Comment (manhnt-0960) [inline config/locales/vi.yml]: STT

**Comment by anhlh-3049** [comment]
ready

**Pull request merged**